docs/usage.md

```diff
@@ -52,12 +50,105 @@ TREATMENT_REP3,AEG588A6_S6_L004_R1_001.fastq.gz,
 
 An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
 
+## Example workflow
```

Comment:
```suggestion
## Example workflow run
```

Reply:
maybe move this section to the end or if you want you can have it in a separate usage docs file (see for example https://nf-co.re/taxprofiler/1.1.8/docs/usage/tutorials)